crates/bevy_transform/src/components/transform.rs

```diff
@@ -527,35 +533,42 @@ impl Default for Transform {
     }
 }
 
-/// The transform is expected to be non-degenerate and without shearing, or the output
-/// will be invalid.
-impl From<GlobalTransform> for Transform {
-    fn from(transform: GlobalTransform) -> Self {
-        transform.compute_transform()
+#[cfg(feature = "bevy-support")]
+mod bevy_support {
+    use std::ops::Mul;
+
+    use super::*;
+
+    /// The transform is expected to be non-degenerate and without shearing, or the output
+    /// will be invalid.
+    impl From<GlobalTransform> for Transform {
+        fn from(transform: GlobalTransform) -> Self {
+            transform.compute_transform()
+        }
     }
-}
 
-impl Mul<Transform> for Transform {
-    type Output = Transform;
+    impl Mul<Transform> for Transform {
```

Comment:
This also shouldn't be feature-gated.

Reply:
The `Mul` impls related to only `Transform` are now not gated

Reply:
Added back the impl